Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppCategories/MSFT_IntuneAppCategories.schema.mof

```diff
@@ -0,0 +1,14 @@
+[ClassVersion("1.0.0.0"), FriendlyName("IntuneAppCategories")]
+class MSFT_IntuneAppCategories : OMI_BaseResource
+{
+    [Key, Description("The unique identifier for an entity. Read-only.")] String Id;
```

Comment:
```suggestion
    [Write, Description("The unique identifier for an entity. Read-only.")] String Id;
```
Id shouldn't be a key property because of cross-tenant sync.